pkgs/development/tools/jq/default.nix

```diff
@@ -9,7 +9,7 @@ stdenv.mkDerivation rec {
     sha256="0g29kyz4ykasdcrb0zmbrp2jqs9kv1wz9swx849i2d1ncknbzln4";
   };
 
-  outputs = [ "bin" "doc" "man" "dev" "lib" "out" ];
+  outputs = [ "out" "doc" "man" "dev" "lib" ];
```

Comment:
I disagree `out` is the default output when unspecified, anything with multiple outputs should name the others accordingly. I don't want to look at the package definition to figure out what the default output is.

```
[ "bin" "lib" ] == [ "out" "lib" ]
[ "lib" "bin" ] == [ "out" "bin" ]
```

Reply:
Another reason not to do this.

```
$ nix-build -A jq
/nix/store/sxp3lykdfknlxkkp7wr0g4jj2xh5cnlz-jq-1.5-bin
$ ./result/bin/jq
zsh: no such file or directory: ./result/bin/jq
$ ls
default.nix  result-bin
```

Reply:
Well that's just how all the other multiple-output packages currently work. In retrospect, it would have been good to be able to say e.g. `outputs = [ "bin" "doc" "man" "dev" "lib" ];` i.e. no `out` anywhere, but this is how things are now.

Typically you don't need to know the name of the default output (except your `nix-build` example), because `${jq}/bin/jq` and `buildInputs = [ jq ];` both work.

Reply:
No `out` anywhere: you typically need to pass some `$PREFIX`...

Reply:
Please see https://nixos.org/nixpkgs/manual/#idm140737317822720, and `4.4.1` especially.

Reply:
Yeah, I still disagree but there seems to be consensus so I'll drop this part.